report/sections/data.tex

```diff
@@ -1,19 +1,29 @@
 \section{Data}
 
-\todo{TODO: Introduction to dataset, where does it come from?, number of files, structure, lm2, lm3, btn, labels, features, imbalance, etc}
+% Introduction to the dataset
+% What are facial landmarks?
+Key points on the face which allow to perform tasks focused on the face, such as emotion classification, are defined as facial landmarks. These facial landmarks (features), for instance the location of the outer left eye brow, are provided in the Bosphorus Database dataset in three different types of files. Files with an `.lm2' extension have 2D feature coordinates, `.lm3' have 3D feature coordinates, and finally the `.bnt' files have both 2D coordinates and corresponding 3D image coordinates, each labeled with the corresponding emotion it represents.
+
+% How many samples?
+% How many labels?
+% Distribution of labels in dataset
+There are a total of 453 samples, each with an `.lm2', `.lm3', and `.bnt' file. A single file corresponds to one out of the six possible emotions (anger, disgust, fear, happy, sadness, and surprise). Of these 453 samples, 71 correspond to `anger', 69 to `disgust', 70 to `fear', 106 to `happy', 66 to `sadness', and 71 to `surprise'. Since some labels have more samples than others, the dataset suffers from class imbalance. Class imbalance is a phenomenon which usually results in models which tend to focus on the classification of the samples which are overrepresented while ignoring or misclassifying the underrepresented samples \cite{data-mining-intro}. Section \ref{impl} explains how this issue was addressed.
```

Comment:
- `\textit`  each emotion in the first set of parentheses?
- [ ] "...is a phenomenon **that** usually"
- [ ] "...while ignoring [emdash --]or misclassifying[emdash] the..."